docs/content/reference/dynamic-configuration/traefik.containo.us_middlewares.yaml

```diff
@@ -91,8 +91,25 @@ spec:
               circuitBreaker:
                 description: CircuitBreaker holds the circuit breaker configuration.
                 properties:
+                  checkPeriod:
+                    description: CheckPeriod is the interval between successive checks
+                      of the circuit breaker condition (standby state).
+                    format: int64
+                    type: integer
                   expression:
+                    description: Expression is the condition that triggers the tripped
+                      state.
                     type: string
+                  fallbackDuration:
+                    description: FallbackDuration is the duration for which the circuit
+                      breaker will wait before trying to recover (tripped state).
+                    format: int64
+                    type: integer
+                  recoveryDuration:
+                    description: RecoveryDuration is the duration for which the circuit
+                      breaker will try to recover (recovering state).
+                    format: int64
```

Comment:
I think these should be "anyOfs" like https://github.com/traefik/traefik/blob/fe684de5983d10efc93a1bb4f4e9b50f736e41e8/docs/content/reference/dynamic-configuration/traefik.containo.us_middlewares.yaml#L553-L557

as they can take duration strings as well as a nanosecond int.

Reply:
(I'm obviously happy to add a commit accordingly, but I don't know if that's going to break your process)

Reply:
Right, we will that 👍